src/transactions/CreateAccountOpFrame.cpp

```diff
@@ -165,8 +165,8 @@ CreateAccountOpFrame::doApply(AppConnector& app, AbstractLedgerTxn& ltx,
     {
         Asset native(ASSET_TYPE_NATIVE);
         opEventManager.newTransferEvent(
-            native, accountToSCAddress(getSourceAccount()),
-            accountToSCAddress(mCreateAccount.destination),
+            native, makeMuxedAccountAddress(getSourceAccount()),
```

Comment:
I don't quite understand what's going on in most of the ops, why do we switch source to be muxed almost everywhere?

Reply:
This is just a rename of the method to `makeMuxedAccountAddress`.